rss_service.py

```
@@ -0,0 +1,234 @@
+                    kind="nitter",
+                    handle=f"@{handle}",
+                )
+            )
+    for url in os.getenv("rss_feed_urls", "").split(","):
+        if url.strip():
+            sources.append(FeedSource(url.strip()))
+    return sources
+
+
+def parse_feed(content: bytes, source: FeedSource) -> list[FeedEntry]:
+    root = ET.fromstring(content)
```

Ruff (0.15.20)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)